test/provision/rpm.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -ex
 /vagrant/provision/generated/common.sh
 
-wget -q -O /etc/yum.repos.d/jenkins.repo @@RPM_URL@@/@@ARTIFACTNAME@@.repo
-rpm --import @@RPM_URL@@/jenkins-ci.org.key
+wget -q -O /etc/yum.repos.d/@@ARTIFACTNAME@@.repo @@RPM_URL@@/@@ARTIFACTNAME@@.repo
```

Comment:
well I need to change the docs for the key anyway, and it is the same documentation!